### ISSUE TITLE
 Identity _userManager.IsInRoleAsync is always false 

### ISSUE THREAD

**Issue by DamienDoumer** [matteofabbri/AspNetCore.Identity.Mongo#30]
Here is how I instantiate the user manager and the role manager:

```
public AdministratorController(IUrlHelper urlHelper,
        UserManager<AppUser> userManager,
        RoleManager<MongoRole> roleManager) : base(urlHelper)
    {
        _userManager = userManager;
        _roleManager = roleManager;
    }
```
Here is how I add a user to a role (it works well):

```
var res1 = await _userManager.AddClaimAsync(user, new Claim(ClaimTypes.Role, roleAddModel.RoleName));
            var res = await _userManager.AddToRoleAsync(user, roleAddModel.RoleName);
```

And after adding a user to a role then going to the database to test if this user exists, it works well. But, when I use this method to test if the user was assigned a specific role, I get a "FALSE" response all the time:

`var r = await _userManager.IsInRoleAsync(user, roleAddModel.RoleId);`

Or even when I try to remove a user from a role, I always get an error stating the user is not assigned to the role. Whereas, the user is assigned and I can see it in the database.

`var res = await _userManager.RemoveFromRoleAsync(user, roleAddModel.RoleName);`

**Comment by matteofabbri** [comment]
Yes, you found a bug. Fixed now in 6.1.3 will be available in minutes :) 
Thanks

**Comment by DamienDoumer** [comment]
Awesome, thanks
